ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -349,6 +349,9 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].pim.ipv4.sparse_mode is arista.avd.defined(true) %}
    pim ipv4 sparse-mode
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].pim.ipv4.dr_priority is arista.avd.defined(true) %}
```

Comment:
I don't think it would work since the dr_priority is a number, not a boolean. If you change it to ...is arista.avd.defined it will work i think. Same on port-channels and vlan interfaces.